packages/plugins/GeoLocation/src/store/getters.ts

```diff
@@ -8,6 +8,11 @@ const getters: PolarGetterTree<GeoLocationState, GeoLocationGetters> = {
   boundaryLayerId: (_, __, ___, rootGetters): string | undefined => {
     return rootGetters?.configuration?.geoLocation?.boundaryLayerId
   },
+  boundaryOnError: (_, __, ___, rootGetters): string | undefined => {
```

Comment:
```suggestion
  boundaryOnError: (_, __, ___, rootGetters) => {
```
Doesn't have to be defined as `PolarGetterTree` already handles this